mysql-test/suite/innodb/t/change_buffer_free.test

```diff
@@ -0,0 +1,53 @@
+#
+# MDEV 33431: Latching order violation reported fil_system.sys_space.latch and ibuf_pessimistic_insert_mutex
+#
+--source include/have_innodb.inc
+--source include/have_debug.inc
```

Comment:
While normally `defined ENABLED_DEBUG_SYNC` is equal to `!defined DBUG_OFF`, it is possible to enable them independently. Please add also `have_debug_sync.inc`.

Reply:
done.